plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/PartitionTransforms.java

```diff
@@ -589,6 +563,20 @@ private static ColumnTransform voidTransform(Type type)
                 block -> new RunLengthEncodedBlock(nullBlock, block.getPositionCount()));
     }
 
+    private static Block transformBlock(Type sourceType, FixedWidthType resultType, Block block, LongUnaryOperator function)
```

Comment:
The commit message says you'd call this `longBlockTransform`

Reply:
pushed a squash. hopefully won't forget to update